src/containers/FormikForm/formikConceptHooks.tsx

```diff
@@ -96,7 +99,9 @@ export function useFetchConceptData(conceptId: number, locale: string) {
   ) => {
     const newConcept = dirty
```

Comment:
Just in case, sett eksplisitt type på `newConcept` siden det er en union av to typer

Reply:
UpdateConcept transformerer typen før den returnerer, så begge kallene vil returnere `ConceptType`.